src/Features/VisualBasic/Portable/CodeCleanup/VisualBasicCodeCleanupService.vb

```diff
@@ -0,0 +1,175 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Option Explicit On
+Option Infer Off
+Option Strict On
+
+Imports System.Collections.Immutable
+Imports System.Composition
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.CodeCleanup
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Formatting
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.Internal.Log
+Imports Microsoft.CodeAnalysis.RemoveUnnecessaryImports
+Imports Microsoft.CodeAnalysis.VisualBasic.RemoveUnusedVariable
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CodeCleanup
+
+    <ExportLanguageService(GetType(ICodeCleanupService), LanguageNames.VisualBasic), [Shared]>
+    Friend Class VisualBasicCodeCleanupService
+        Implements ICodeCleanupService
+        Private ReadOnly _codeFixServiceOpt As ICodeFixService
+
+        <ImportingConstructor>
+        <Obsolete(MefConstruction.ImportingConstructorMessage, True)>
+        Public Sub New(
+            <Import(AllowDefault:=True)> codeFixService As ICodeFixService _ ' will remove the AllowDefault once CodeFixService is moved to Features
+ _ ' https://github.com/dotnet/roslyn/issues/27369
+)
```

Comment:
i can't read these comments.  please move this comment below or above or something.

Reply:
removed based in issue being fixed